src/cloud-api-adaptor/test/provisioner/provision.go

```diff
@@ -368,6 +381,22 @@ func (p *KeyBrokerService) GetKbsEndpoint(ctx context.Context, cfg *envconf.Conf
 	return "", fmt.Errorf("Service %s not found", serviceName)
 }
 
+func (p *KeyBrokerService) EnableKbsAllowAllPolicy(kbsEndpoint string) error {
+	log.Info("EnableKbsAllowAllPolicy")
+	kbsClientDir := filepath.Join(TRUSTEE_REPO_PATH, "target/release")
+	privateKey := "../../kbs/config/kubernetes/base/kbs.key"
+	policyFile := "../../kbs/sample_policies/allow_all.rego"
```

Comment:
I don't think we should fall back to the allow_all policy for all TEEs. AFAIU the `sample` TEE is excluded from the default policy b/c a failing attester would fall back to the sample attester. In our case a test would erroneously pass, even if a real TEE fails to produce HW evidence.

Ideally we create a policy that will allow only the TEE we are testing for

Reply:
I got the concerns, maybe, we can change it to use a customized policy on demand. I'll introduce a new variable like `CUSTOMIZED_TEE_OPA=${opa_file_path}` to keep the e2e restriction and bring possible more wide test cases.